### PR TITLE
Icons fontawesome

### PR DESCRIPTION
**Pull request by pgiraud**
The goal of this pull request is to use fontawesome as much as possible.
For icônes not available in fontawesome, we use our own font. It comes with documentation on how to build it when we want to add new icons.

Please review.

### CODE REVIEW

**Comment by pgiraud** [comment]
The result can ben seen here: http://pgiraud.github.io/ngeo/icons_fontawesome/examples/contribs/gmf/apps/mobile/index.html

**Comment by ebelo** [comment]
Looks good, thanks!

File: contribs/gmf/fonts/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+In order to do so, you can add new glyphs to the `svg` file.
+
+Then the `.ttf`, `.woff` and `.eot` files should be generated with the
+following commands:
+
+```
+node_modules/svg2ttf/svg2ttf.js contribs/gmf/fonts/gmf-icons.svg contribs/gmf/fonts/gmf-icons.ttf
+node_modules/ttf2eot/ttf2eot.js contribs/gmf/fonts/gmf-icons.ttf contribs/gmf/fonts/gmf-icons.eot
+node_modules/ttf2woff/ttf2woff.js contribs/gmf/fonts/gmf-icons.ttf contribs/gmf/fonts/gmf-icons.woff
```
Comment (sbrunner) [inline contribs/gmf/fonts/README.md]: I should be done in the Makefile ...

File: Makefile [hunk 2]
```diff
@@ -386,6 +386,7 @@ node_modules/angular/angular.min.js: .build/node_modules.timestamp
 .build/node_modules.timestamp: package.json
 	npm install
 	mkdir -p $(dir $@)
+	cp node_modules/font-awesome/fonts/* contribs/gmf/fonts
```
Comment (sbrunner) [inline Makefile]: It should be a separate rule...

**Comment by sbrunner** [comment]
I think that the icons are too small ...
I don't find the localization icon significant, but I don't know what the PSC want's ...

**Comment by ybolognini** [comment]
Nice, but yes we could test with larger icons.
As for localization, let's keep it like that for now.

File: Makefile (hunk 2 above)
Comment (pgiraud) [inline Makefile]: Fixed. I hope this is correct.

File: contribs/gmf/fonts/README.md (hunk 1 above)
Comment (pgiraud) [inline contribs/gmf/fonts/README.md]: Done.

**Comment by pgiraud** [comment]
The icons are too small, I agree. I'll give it a new try in a new pull request while cleaning up and reorganizing less files.

**Pull request merged**